Packages/com.unity.multiplayer.samples.coop/Utilities/Net/SessionManager.cs

```diff
@@ -56,7 +67,8 @@ public void DisconnectClient(ulong clientId)
                 // Mark client as disconnected, but keep their data so they can reconnect.
                 if (m_ClientIDToPlayerId.TryGetValue(clientId, out var playerId))
                 {
-                    if (GetPlayerData(playerId)?.ClientID == clientId)
+                    var playerData = GetPlayerData(playerId);
+                    if (playerData != null && playerData.Value.ClientID == clientId)
```

Comment:
shouldn't this be `if (playerData != null && playerData.ClientID == clientId)` instead?
In the previous version, the `.Value` wasn't accessed explicitly.